src/energyplus/CMakeLists.txt

```diff
@@ -200,6 +200,7 @@ set(${target_name}_src
   ForwardTranslator/ForwardTranslateGeneratorFuelCellStackCooler.cpp
   ForwardTranslator/ForwardTranslateGeneratorFuelCellWaterSupply.cpp
   ForwardTranslator/ForwardTranslateGeneratorFuelSupply.cpp
+  ForwardTranslator/ForwardTranslateGeneratorWindTurbine.cpp
```

Comment:
ReverseTranslator?

Reply:
@joseph-robertson As a reminder, GenerateClass.rb will generate one for you that should require minimal changes.

Full command
```
cd developer/ruby/
$os_build/Products/openstudio GenerateClass.rb -c "GeneratorWindTurbine" -b "Generator"  -i "OS:Generator:WindTurbine" -s model -o /home/julien/Software/Others/OpenStudio/src/model/ -p -f -r
```

I think this should only do the RT:

```
cd developer/ruby/
$os_build/Products/openstudio GenerateClass.rb -c "GeneratorWindTurbine" -b "Generator"  -i "OS:Generator:WindTurbine" -r
```